apps/condo/domains/billing/access/BillingIntegrationLog.js

```diff
@@ -11,20 +11,19 @@ async function canReadBillingIntegrationLogs ({ authentication: { item: user } }
     if (!user) return false
     if (user.isAdmin) return true
     return {
-        // TODO(pahaz): wait https://github.com/keystonejs/keystone/issues/4829 (no access check!)
-        // context: {
-        //     OR: [
-        //         {
-        //             organization: {
-        //                 employees_some: {
-        //                     user: { id: user.id },
-        //                     role: { canManageIntegrations: true },
-        //                 },
-        //             },
-        //         },
-        //         { integration: { accessRights_some: { user: { id: user.id } } } },
-        //     ],
-        // },
+        context: {
+            OR: [
+                {
+                    organization: {
+                        employees_some: {
+                            user: { id: user.id },
+                            role: { canManageIntegrations: true },
```

Comment:
@toplenboren you should know whats going wrong here 😉

Reply:
Fixed!

Reply:
Using OR in access checks may extend results - better to use AND
We have the same bug when I wrote OR to the propery access: when I asked properties from 1 organization - access resolver extended the result to all organizations where I am employees_some
And, also, you do not check that employee is blocked or not

Reply:
Do we need to check
```
deletedAt: null
```

Reply:
Thank you for review!

I added the check for deletedAt and isBlocked, but I do not agree on the OR point. We need to use OR, because we have two mutual excluding uses who can access this lad